examples/batch_processing.ipynb

```diff
@@ -0,0 +1,1148 @@
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CAPTION: A set of two black leather upholstered dining chairs with a simple, contemporary design.\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<img src=\"https://m.media-amazon.com/images/I/41RgefVq70L._SS522_.jpg\"/>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CAPTION: A green, waterproof, square plant repotting mat with raised corners and gardening tools displayed on it.\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<img src=\"https://m.media-amazon.com/images/I/61vz1IglerL._SS522_.jpg\"/>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CAPTION: A brown absorbent non-slip doormat with the phrase \"it's a good day to play PICKLEBALL\" and pickleball paddle graphics.\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<img src=\"https://m.media-amazon.com/images/I/41p4d4VJnNL._SS522_.jpg\"/>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CAPTION: A set of four grey foldable TV trays with a matching stand, featuring a sleek and space-saving design for convenient snacking or dining.\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Testing on a few images\n",
+    "for _, row in df[:5].iterrows():\n",
+    "    img_url = row['primary_image']\n",
+    "    caption = get_caption(img_url, row['title'])\n",
+    "    img = Image(url=img_url)\n",
+    "    display(img)\n",
+    "    print(f\"CAPTION: {caption}\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a89a6709",
+   "metadata": {},
+   "source": [
+    "## Creating the batch file\n",
+    "\n",
+    "The batch file (jsonl) should contain one line per task.\n",
+    "Each task is defined as such:\n",
+    "\n",
+    "```\n",
+    "{\n",
+    "    \"custom_id\": <TASK_ID>,\n",
+    "    \"method\": \"POST\",\n",
+    "    \"url\": \"/v1/chat/completions\",\n",
+    "    \"body\": {\n",
+    "        \"model\": <MODEL>,\n",
+    "        \"messages\": <MESSAGES>,\n",
+    "        // other parameters\n",
+    "    }\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Note: the task ID should be unique per batch job. This is what you can use to match results to the initial input files, are tasks will not be returned in the same order."
```

Comment:
do you mean the ID is unique per each task?

Reply:
Not sure about the wording here. To me each line is a task that is part of a broader batch job. Would you say something different?